tcks/microprofile-health/pom.xml

```diff
@@ -18,6 +18,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
+                    <systemPropertyVariables>
+                        <!-- Change metrics endpoint to be /metrics -->
```

Comment:
Should we consider updating the TCKs to allows setting the prefix?

Reply:
Some TCKs explicitly indicate the name of the URL path, sometimes in relation to root context, so I don't think this would be possible